lib/prettier.js

```diff
@@ -33,35 +33,35 @@ module.exports = {
       this.handleEvents.bind(this)
```

Comment:
I think we can remove this line because `handleEvents` is now bound via fat arrow on line 77